categorize_transactions.py

```diff
@@ -0,0 +1,157 @@
+
+# Category object
+# properties
+# tags = keywords
+# desc = string description of category
+
+categories = {}
```

Comment:
What is this? What are the keys? What are the values? Is this supposed to be a hierarchy of categories? Or are these supposed to be tags?